create_infotexts.py

```diff
@@ -333,5 +230,145 @@ def main():
         outfile.close()
 
 
+class CypernImage():
+    """Process the information for a single image."""
+
+    def __init__(self):
+        """Instantiate a single instance of a processed image."""
+        self.content_cats = []  # content cateogories wihtout 'Category:'-prefix
+        self.meta_cats = []  # maintance categories without 'Category:'-prefix
+        self.data = {}  # dictionary holding individual field values as wikitext
+
+        batch_cats = ["Swedish Cyprus Expedition|Swedish Cyprus Expedition",
+                      "Media_from_the_National_Museums_of_World_Culture",
+                      "Media_contributed_by_SMVK_2017-02"]
+
+        self.meta_cats.extend(batch_cats)
+
+
+    def process_depicted_people(self, names_string):
+        """
+        Create depicted people wikitext from raw input data.
+
+        Populates the data["depicted_people"] field in self and interacts with
+        content_cats and meta_cats.
+        
+        If name interpratation fails, we still store the problematic string + maintanence category.
+
+        :param names_string: string representing one or more names.
+        """
+        try:
+            names = CypernImage.isolate_name(names_string)
+        except ValueError:
+            self.meta_cats.append("Media_contributed_by_SMVK_2017-02_with_faulty_depicted_person_values")
```

Comment:
I believe this should be `Media_contributed_by_SMVK_with_faulty_depicted_person_values` (i.e. tied to the dataset not the batch)